### PR TITLE
Tooltip code

### PR DESCRIPTION
**Pull request by Pranjal26101999**

### CODE REVIEW

File: libs/ptg-ui-react-lib/src/lib/download-file/downloadFile.tsx [hunk 1]
```diff
@@ -149,11 +149,6 @@ const PtgUiDownload = ({
 
   return (
     <div className="container">
-      <div className="row">
-        <div className="col-12">
```
Comment (ZalawaHarsha) [inline libs/ptg-ui-react-lib/src/lib/download-file/downloadFile.tsx:153]: why this change is needed ?

Comment (Pranjal26101999) [reply libs/ptg-ui-react-lib/src/lib/download-file/downloadFile.tsx:153]: to add the toggle button in front of File Download heading i have removed File Download heading from  common component and added that heading with toggle button react-app download.tsx file
